tests/gdx-tests/src/com/badlogic/gdx/tests/IntersectorOverlapConvexPolygonsTest.java

```diff
@@ -0,0 +1,94 @@
+
+    private Polygon triangle1 = new Polygon();
+    private Polygon triangle2 = new Polygon();
+
+    private Intersector.MinimumTranslationVector mtv = new Intersector.MinimumTranslationVector();
+
+    private Vector3 mouseCoords = new Vector3();
+
+    @Override
+    public void create() {
+        Gdx.app.setLogLevel(Application.LOG_DEBUG);
+        camera = new OrthographicCamera();
+        camera.setToOrtho(false);
+        camera.position.set(0, 0, 0);
+        shapeRenderer = new ShapeRenderer();
+        shapeRenderer.setAutoShapeType(true);
+        //set inital position
+        triangle1.setVertices(vertsTriangle1);
+        triangle2.setVertices(vertsTriangle2);
+        triangle1.setPosition(0, 0);
+        triangle2.setPosition(10, 0);
+    }
+
+    @Override
+    public void resize(int width, int height) {
+        camera.viewportWidth = 80;
+        camera.viewportHeight = 60;
+    }
+
+    private void update(float deltaTime) {
+        if (Gdx.input.isKeyJustPressed(Input.Keys.SPACE)) {
+            Intersector.overlapConvexPolygons(triangle1, triangle2, mtv);
+            float x = triangle1.getX() + (mtv.normal.x * mtv.depth);
+            float y = triangle1.getY() + (mtv.normal.y * mtv.depth);
+            triangle1.setPosition(x, y);
+            Gdx.app.debug(TAG, mtv.normal + " " + mtv.depth);
+        }
+        if (Gdx.input.isButtonPressed(Input.Buttons.LEFT)) {
+            mouseCoords.set(Gdx.input.getX(), Gdx.input.getY(), 0);
+            camera.unproject(mouseCoords);
+            triangle1.setPosition(mouseCoords.x, mouseCoords.y);
+            boolean overlaps = Intersector.overlapConvexPolygons(triangle1, triangle2, mtv);
+            Gdx.app.debug(TAG, mtv.normal + " " + mtv.depth + " overlaps: " + overlaps + " " + mouseCoords);
+        } else if (Gdx.input.isButtonJustPressed(Input.Buttons.RIGHT)) {
+            triangle1.rotate(90);
+            boolean overlaps = Intersector.overlapConvexPolygons(triangle1, triangle2, mtv);
+            Gdx.app.debug(TAG, mtv.normal + " " + mtv.depth + " overlaps: " + overlaps);
+        }
+    }
+
+    @Override
+    public void render() {
+        update(Gdx.graphics.getDeltaTime());
+        camera.update();
+        Gdx.gl.glClearColor(0f, 0f, 0f, 0f);
+        Gdx.gl.glClear(GL20.GL_COLOR_BUFFER_BIT | GL20.GL_DEPTH_BUFFER_BIT);
+        shapeRenderer.setProjectionMatrix(camera.combined);
+        shapeRenderer.begin(ShapeRenderer.ShapeType.Line);
+        shapeRenderer.setColor(Color.GOLD);
+        shapeRenderer.polygon(triangle1.getTransformedVertices());
+        shapeRenderer.setColor(Color.CYAN);
+        shapeRenderer.polygon(triangle2.getTransformedVertices());
+        shapeRenderer.setColor(Color.RED);
+        shapeRenderer.line(mtv.normal.x, mtv.normal.y, mtv.normal.x * 10, mtv.normal.y * 10);
+        shapeRenderer.set(ShapeRenderer.ShapeType.Filled);
+        shapeRenderer.circle((mtv.normal.x), (mtv.normal.y), 0.5f);
```

Comment:
The test is great, I don't understand what the circle represents. Wouldn't it be more useful (instead of having that circle) to display the MTV depth with a green line over the current red line. Something like:
`shapeRenderer.line(0, 0, mtv.depth * mtv.normal.x , mtv.depth * mtv.normal.y);`

I would also make the red line start at (0,0).

Reply:
Hello @obigu thanks for your reply. I made the circle to better visualize the direction the line is pointing at. Without the point you would only see a line but it is harder to predict where the start of this line is. Maybe it would be better to draw the red line also in the oposite direction then the dot will become the origin of the line.

Yes you are right, i can translate the position of the line to 0,0. Be careful, the line represents the minimum projecton axis where the points min and max are projected from the shapes and also the direction. Showing the green line as depth is a good idea. At the moment i just show the axis with a length of 10. I will fix that.

Reply:
Thanks. It's a bit better now but why do you substract `-mtv.normal.x` and `mtv.normal.y` to all the lines (red, grey and green) y coordinates?. The idea of the green line is to visually displays the MTV depth, now it goes negative when it's small.

Reply:
Oh yes my bad. I forgot it is a unit vector and not a coordinate in space. Tried translating it (the line) to point 0,0 which of course doesn't make any sense because the vector is just pointing in a direction. Did not see it because i used a larger scale but now i saw it reducing the scale. Thanks for reviewing and mentioning it.